### PR TITLE
docs: create LSP2 page + add figures for LSP6

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduces?

- [x] added more content for LSP2 page
- [x] create figure to explain key type `Singleton`
- [x] create figure to explain key type `Array`
- [x] create figure to explain key type `Mapping`
- [x] create figure to explain key type `Bytes20Mapping`
- [x] create figure to explain key type `Bytes20MappingWithGrouping`
- [x] added more figures in LSP6 (introduction, **Allowed Standards** and permissions `ADDPERMISSIONS` + `CHANGEPERMISSIONS`

### CODE REVIEW

File: docs/standards/generic-standards/01-lsp2-json-schema.md [hunk 1]
```diff
@@ -5,51 +5,117 @@ sidebar_position: 3.3
...
+By introducing a schema, contract storage can be represented in the same way across contracts in the network. Everyone has a unified view of the data stored between smart contracts. Data can then be easily parsed, and contracts or interfaces can read or write data from/to the contract storage in the same manner. This makes smart contracts **more interoperable with each other**.
 
-LSP2 enables to store information on an ERC725Account in an organized and unified way.
+### How does LSP2 work?
 
-By introducing a new storage schema for smart contracts, ERC725Accounts become interroperable between each other, as their storage is represented in the same way. Contract and interfaces can then all read and write to the storage in the same manner. Data can be easily parsed, so that any smart contract or interface can interact with them and fetch data.
+LSP2 introduces new ways to encode data, depending on its type. From a single entry, to multiple entires (like arrays or maps).
```
Comment (Hugoo) [inline docs/standards/generic-standards/01-lsp2-json-schema.md:51]:
```suggestion
LSP2 introduces new ways to encode data, depending on its type. From a single entry, to multiple entries (like arrays or maps).
```

File: docs/standards/universal-profile/01-lsp0-erc725account.md [hunk 2]
```diff
@@ -48,25 +49,23 @@ This is very useful for accounts where anyone could customize the way his accoun
 
 Check **[LSP1-UniversalReceiver](../generic-standards/02-lsp1-universal-receiver.md)** standard for more information.
 
-
 ### ERC1271
 
-Externally Owned Accounts (EOA) can sign messages with their associated private keys, but currently contracts cannot. This standard defines a way for contracts to verify if a provided signature is valid when the account is a smart contract.                  
+Externally Owned Accounts (EOA) can sign messages with their associated private keys, but currently contracts cannot. This standard defines a way for contracts to verify if a provided signature is valid when the account is a smart contract.
```
Comment (Hugoo) [inline docs/standards/universal-profile/01-lsp0-erc725account.md:54]:
```suggestion
Externally Owned Accounts (EOAs) can sign messages with their associated private keys, but contracts cannot. This standard defines a way for contracts to verify if a provided signature is valid when the account is a smart contract.  
```

File: docs/standards/universal-profile/04-lsp6-key-manager.md [hunk 3]
```diff
@@ -191,19 +196,29 @@ You can find **more details about each permissions by clicking on the toggles be
...
+    <summary><code>CHANGEPERMISSIONS</code> - Allows changing existing permissions of addresses</summary>
     <p style={{marginBottom: '3%', marginTop: '2%', textAlign: 'center'}}>
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000000002</code>
     </p>
-    <p>This permission allows an address to <b>grant + revoke</b> permissions for any specific address (including itself).</p>
+    <p>This permission allows to <b>edit permissions</b> of any address that has already some permissions set on the ERC725Account (including itself).</p>
+
+![CHANGE Permissions](./../../../static/img/standards/lsp6-change-permissions.jpeg)
+
+<p>Bear in mind that the behaviour of CHANGEPERMISSIONS slightly varies, depending on the new permissions value being set (see figure below).</p>
```
Comment (Hugoo) [inline docs/standards/universal-profile/04-lsp6-key-manager.md:207]:
```suggestion
<p>Bear in mind that the behaviour of `CHANGEPERMISSIONS` slightly varies, depending on the new permissions value being set (see figure below).</p>
```

Not sure if the MD syntax works in this HTML part but maybe put the "code" syntax to this word would be nice?

**Review by Hugoo — APPROVED** [review]
Cool !

**Review by YamenMerhi — APPROVED** [review]

**Pull request merged**